interface/wx/grid.h

```diff
@@ -4409,6 +4577,9 @@ class wxGrid : public wxScrolledCanvas
         a grid with a million of columns, we don't want to create an array with
         a million of entries in this function, instead it returns an empty
         array and GetSelectedCols() returns an array containing one element).
+
+        @since 3.1.4 The function can be slow for the big grids, use
+        GetSelectionIterator in the new code.
```

Comment:
I think it needs parentheses to be auto-linked in the generated docs.
```suggestion
        GetSelectionIterator() in the new code.
```